### DISCUSSION TITLE
在插件中，rewrite和header_filter两个阶段都能被执行到吗

### DISCUSSION THREAD

**Discussion by FinerKeysen** [apache/apisix#7649] [General]
可不可以在一个插件中同时完成对请求header（_M.rewrite）和响应header（_M.header_filter）的修改？两个阶段都能被执行到吗？

**Comment by tokers** [comment]
Yes. And before you ask here, please read the document first: https://apisix.apache.org/zh/docs/apisix/plugin-develop/#apisix-%E7%9A%84%E8%87%AA%E5%AE%9A%E4%B9%89%E9%98%B6%E6%AE%B5

**Reply by FinerKeysen** [reply]
thx~😁, i'll make a try.
